packages/manager/modules/user-contracts/src/template.html

```diff
@@ -0,0 +1,47 @@
+<div class="container-fluid my-5 hub-main-view_container">
```

Comment:
`hub-main-view_container` css should be replicated in this module if it needs to be used :)

Reply:
Have removed this